packages/perseus/src/widgets/number-line/score-number-line.ts

```diff
@@ -1,44 +1,41 @@
 import {number as knumber} from "@khanacademy/kmath";
 
+import validateNumberLine from "./validate-number-line";
+
 import type {PerseusScore} from "../../types";
 import type {
-    PerseusNumberLineRubric,
+    PerseusNumberLineScoringData,
     PerseusNumberLineUserInput,
 } from "../../validation.types";
 
 function scoreNumberLine(
-    state: PerseusNumberLineUserInput,
-    rubric: PerseusNumberLineRubric,
+    userInput: PerseusNumberLineUserInput,
+    scoringData: PerseusNumberLineScoringData,
 ): PerseusScore {
-    const range = rubric.range;
-    const divisionRange = state.divisionRange;
-    const start = rubric.initialX != null ? rubric.initialX : range[0];
-    const startRel = rubric.isInequality ? "ge" : "eq";
-    const correctRel = rubric.correctRel || "eq";
+    const validationError = validateNumberLine(userInput);
+    if (validationError) {
+        return validationError;
+    }
+
+    const range = scoringData.range;
+    const start =
+        scoringData.initialX != null ? scoringData.initialX : range[0];
+    const startRel = scoringData.isInequality ? "ge" : "eq";
+    const correctRel = scoringData.correctRel || "eq";
     const correctPos = knumber.equal(
-        state.numLinePosition,
-        rubric.correctX || 0,
+        userInput.numLinePosition,
+        scoringData.correctX || 0,
     );
-    const outsideAllowedRange =
-        state.numDivisions > divisionRange[1] ||
-        state.numDivisions < divisionRange[0];
 
-    // TODO: I don't think isTickCrtl is a thing anymore
-    if (state.isTickCrtl && outsideAllowedRange) {
-        return {
-            type: "invalid",
-            message: "Number of divisions is outside the allowed range.",
-        };
-    }
-    if (correctPos && correctRel === state.rel) {
+    if (correctPos && correctRel === userInput.rel) {
         return {
             type: "points",
             earned: 1,
             total: 1,
             message: null,
         };
     }
-    if (state.numLinePosition === start && state.rel === startRel) {
+    if (userInput.numLinePosition === start && userInput.rel === startRel) {
```

Comment:
This check could possibly be moved to the validation function at a later time if the flow can change so this check occurs first. Only possible if the answer is never the starting position though.